dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -177,7 +177,7 @@ private String getRepositoryUrl(String artifactId) {
 
   public class Version implements Comparable<Version> {
 
-    private String version;
+    public String version;
 
     @JsonValue
     public final Map<String, String> getJson() {
```

Comment:
If what you are trying to do is get `{"version":"x"}`, then just the following is needed:

```java
@Value
public class Version
  private String version;
}
```

No need for all the other stuff